### PR TITLE
Update instructions for installing Python 3.8.15 instead of Python 3.8.12

### PR DESCRIPTION
**Pull request by vojtechjelinek**
Update instructions for installing Python 3.8.15 instead of Python 3.8.12.

Connected to https://github.com/oppia/oppia/pull/16730 and https://github.com/oppia/release-scripts/pull/112

### CODE REVIEW

File: Installing-Oppia-(Linux;-Python-3).md [hunk 1]
```diff
@@ -389,18 +389,18 @@ On Arch Linux, you should follow these instructions to set up your virtual envir
 4. Now you can install Python 3.7.10 and the associated pip like this:
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md]: 3.7.10 -> 3.8.15

File: Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md [hunk 2]
```diff
@@ -1,29 +1,42 @@
-# Upgrading python interpreter version from 3.7.10 to 3.8.12
+# Upgrading Python interpreter from any version to version 3.8.15
 
-Oppia changed the Python version it uses from 3.7.10 to 3.8.12 in Oct 2022. This page is aimed at developers who are currently using Python 3.7.10 and need to upgrade to 3.8.12. 
-
-For effective and effortless switching to the newer version, this guide will provide a brief and detailed instruction on how to achieve just that. Please follow the below mentioned instructions carefully.
+For effective and effortless switching to the newer version, this guide will provide a brief and detailed instruction on how to achieve just that. Please follow the instructions below carefully.
```
Comment (U8NWXD) [inline Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md]:
```suggestion
This guide will walk you through upgrading Python. Please follow the instructions below carefully.
```

I think the extra words are mostly superfluous

File: Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md [hunk 3]
```diff
@@ -57,18 +70,18 @@ $ direnv reload
 
 1. In this step, we will set up a new virtual enviornment for our new python interpreter to specifically work with Oppia.
 
-    Use the command given below to set up a virtualenv with python `3.8.12`. Be sure to replace the `{unique-virtualenv-name}` with a suitable virtualenv name:
+    Use the command given below to set up a virtualenv with python `3.8.15`. Be sure to replace the `{unique-virtualenv-name}` with a suitable virtualenv name:
 
     ```
-    pyenv virtualenv 3.8.12 {unique-virtualenv-name}
+    pyenv virtualenv 3.8.15 {unique-virtualenv-name}
```
Comment (U8NWXD) [inline Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md:76]: Shouldn't this now use direnv?

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md (hunk 3 above)
Comment (vojtechjelinek) [reply Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md:76]: Yeah, I think I can also remove this part, but I feel like it is better to preserve it if there are people that still use just pyenv.

File: Installing-Oppia-(Linux;-Python-3).md (hunk 1 above)
Comment (vojtechjelinek) [reply Installing-Oppia-(Linux;-Python-3).md]: Done, also replaced other references.

File: Google-Summer-of-Code-2022.md [hunk 4]
```diff
@@ -116,7 +116,7 @@ Welcome! If you're interested in applying to work with Oppia for GSoC, please fo
 
 **Q: What technical skills do I need to work on Oppia?**
 
-A: For Oppia Web, Angular 2+, Python 3.7 and Google App Engine is useful and recommended for most Oppia work; in addition, UI design skills are useful for frontend, user-facing work. For Oppia Android, you'll want to be familiar with Android development in Kotlin. Please see the individual project ideas to determine whether these skills are recommended for the project in question.
+A: For Oppia Web, Angular 2+, Python 3.8 and Google App Engine is useful and recommended for most Oppia work; in addition, UI design skills are useful for frontend, user-facing work. For Oppia Android, you'll want to be familiar with Android development in Kotlin. Please see the individual project ideas to determine whether these skills are recommended for the project in question.
```
Comment (U8NWXD) [inline Google-Summer-of-Code-2022.md]: I don't think we want to change archived GSoC pages, right? This should reflect what contributors actually saw during GSoC 2022

**Review by U8NWXD — CHANGES_REQUESTED** [review]
Also there's one comment left from before: https://github.com/oppia/oppia-web-developer-docs/pull/66#discussion_r1050009292

File: Google-Summer-of-Code-2022.md (hunk 4 above)
Comment (vojtechjelinek) [reply Google-Summer-of-Code-2022.md]: Makes sense, removed the change.

File: Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md (hunk 2 above)
Comment (vojtechjelinek) [reply Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md]: Done.

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**